src/frontends/common/src/manager.cpp

```diff
@@ -18,6 +18,9 @@ using namespace ov::frontend;
 class FrontEndManager::Impl {
     std::mutex m_loading_mutex;
     std::vector<PluginInfo> m_plugins;
+    // List of predefined plugins file names (without prefix and suffix)
+    const std::vector<std::string> m_plugins_files_names = {"ir", "onnx", "tensorflow", "paddle",
+                                         /*test frontends*/ "mock_py", "mock1", "mock_mo"};
```

Comment:
not sure that hardcoding tests FEs is a good idea. It creates vulnerability since external users can create such FEs and they will be mistakenly loaded

Reply:
I agree, but it was done in order to save compatibility with all known plugins. Otherwise we need to provide API to load custom plugins